sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -375,6 +375,7 @@ func (m *marshaller) addMultiEdge(edge NamedEdge) []string {
 		payload := &pipepb.WindowIntoPayload{
 			WindowFn: makeWindowFn(edge.Edge.WindowFn),
 		}
+		transformEnvID = m.addDefaultEnv()
```

Comment:
This should be native as well, right ?

Reply:
No, `Window.into`requires execution of `assignWindows` that is part of the windowing fn.

It could be lifted into the runner if it understands the windowing fn but that could break fusion since `Window.into` may occur between `ParDo`s and far away from `GroupByKey`.

Reply:
Ah ok. I read this as GBK by mistake. Can we do this in a common place and skip the two known runner implemented transforms similar to other SDKs ?

Reply:
Done